lib/usage.dart

```diff
@@ -297,10 +297,7 @@ String sanitizeStacktrace(dynamic st, {bool shorten: true}) {
 
   if (shorten) {
     // Shorten the stacktrace up a bit.
-    str = str
-        .replaceAll('(package:', '(')
-        .replaceAll('(dart:', '(')
-        .replaceAll(new RegExp(r'\s+'), ' ');
+    str = str.replaceAll(new RegExp(r'[\t ]+'), ' ');
```

Comment:
Could this be hit more than a few times? May want to make the `RegExp` an instance or top-level static var